engineapi/engineapi/routes/metatx.py

```diff
@@ -414,3 +431,32 @@ async def delete_requests(
         raise EngineHTTPException(status_code=500)
 
     return deleted_requests
+
+
+# @app.post("/requests/{request_id}/complete", tags=["requests"])
+@app.post("/requests/complete", tags=["requests"])
```

Comment:
This should be `/requests/<request_id>/complete` IMO

Reply:
The problem in `whitelist_paths`, dynamic `<request_id>` and our current implementation of authentification middleware. I tried to avoid making this over complicated changes so this is the only way to make this route public.

Reply:
How complex would it be to update our `whitelist_paths` implementation?